quickstart/k8s-the-prod-way/infrastructure.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Provisioning Shared, Managed Infrastructure
+---
+
+This lab assumes you have successfully completed [lab 1](./identity.html), the results of which are
+to have successfully bootstrapped the identity platform for CI/CD and teams.
+
+In this tutorial, we will use this identity to:
+
+* Create a shared, managed Kubernetes cluster
+* Create a shared, managed SQL database
+
+Inside the [Kubernetes the Prod Way repository][ktpw], the `infrastructure` stack is implemented for
+each of [AWS][aws], [Azure][azure], and [GCP][gcp].
+
+## Prerequisites: Logging into cloud provider with CLI
+
+In the first lab, we need to set up credentials using the CLI, so that Pulumi can authenticate
+against the cloud provider of your choice. Make sure you're still logged in before continuing.
+
+## Provisioning
+
+Begin by `cd`'ing into the `infrastructure` directory of the cloud provider of your choice. From
+there, install the Pulumi toolchain:
+
+```sh
+yarn install
+```
+
+The infrastructure stack comes with a script that will allow you to authenticate using the CI/CD
+service account we defined in lab 1. We will use this to provision this cluster -- later we will use
+this in CI/CD.
+
+Assuming you have successfully finished lab 1, you'll need to specify the name of the identity stack
+you've created. This was the argument you passed to `pulumi stack init`.
+
+```sh
+./scripts/login-to-ci-service-account.sh <identity-stack-name>
```

Comment:
Honestly wish all of this could be bash free.

Reply:
Seeing all the commands helps the reader understand how everything fits together.

Reply:
We can show them what's _inside_ the scripts, but I do not want to get rid of them because they're genuinely very useful.

Reply:
Where are they? I must have missed them. https://github.com/pulumi/kubernetes-the-prod-way doesn't seem to have a `scripts` directory, is that where they are?